jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/Annotator.java

```diff
@@ -160,6 +160,20 @@
      *         properties'.
      */
     boolean isAdditionalPropertiesSupported();
+    
+    /**
+     * Add the <code>@JsonFormat</code> annotation to mark a java.util.Date field
+     * 
+     * @param field
+     *            the field that contains data that will be serialized
+     * @param clazz
+     *            the owner of the field (class to which the field belongs)
+     * @param propertyName
+     *            the name of the JSON property that this field represents
+     * @param propertyNode
+     *            the schema node defining this property
+     */
+    void jsonFormat(JFieldVar field, JDefinedClass clazz, String propertyName, JsonNode node);
```

Comment:
This method needs a new name I think, `jsonFormat` doesn't describe what this does, it's too general. I think a good name for this method (that fits with the other methods in this interface) would be: `dateField`.